exonum/src/helpers/types.rs

```diff
@@ -113,6 +115,21 @@ impl Height {
     }
 }
 
+impl BinaryValue for Height {
+    fn to_bytes(&self) -> Vec<u8> {
+        self.0.into_bytes()
+    }
+
+    fn from_bytes(value: Cow<'_, [u8]>) -> Result<Self, failure::Error> {
+        Ok(Self(
+            <u64 as BinaryValue>::from_bytes(value).expect("Error while deserializing value"),
```

Comment:
Why does this method panic? `BinaryValue` deals with untrusted input, so this looks inappropriate.

Reply:
I just copy-pasted an implementation from `Round` several lines below :slightly_smiling_face:.
I'll fix it there as well then.